src/sysctl.cr

```diff
@@ -0,0 +1,87 @@
+lib LibC
```

Comment:
@ysbaddaden, The syscall is deprecated on Linux but it's not available through the libc anymore, so no risks there. So your suggestion is to write those by hands until it is supported by the generator, am I right ?